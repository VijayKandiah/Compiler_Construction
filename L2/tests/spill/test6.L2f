(:myF
  0 2

  mem rsp 8 <- 3

  %myVar <- 0
  mem rsp 0 <- %myVar


  %myVar <- mem rsp 8

  return
)

%myVar

%S
