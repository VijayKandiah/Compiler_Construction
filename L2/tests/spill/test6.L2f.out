(:myF
	0 3
	mem rsp 8 <- 3
	%S0 <- 0
	mem rsp 16 <- %S0
	%S1 <- mem rsp 16
	mem rsp 0 <- %S1
	%S2 <- mem rsp 8
	mem rsp 16 <- %S2
	return
)
